src/lib/statuses.js

```diff
@@ -48,6 +48,7 @@ const statuses = {
   444: { code: 444, message: 'No Response' },
   450: { code: 450, message: 'Blocked by Windows Parental Controls' },
   451: { code: 451, message: 'Unavailable For Legal Reasons' },
+  497: { code: 497, message: 'HTTP Request sent to HTTPS port' },
```

Comment:
Sorry, should have seen this before. Can you also update the capitalization in the image text? Should be ready to merge right after.

```suggestion
  497: { code: 497, message: 'HTTP Request Sent to HTTPS port' },
```

Reply:
Alright, just give me one minute :)